administracion/forms.py

```diff
@@ -42,21 +41,55 @@ class FormaCreacionUsuario(FormaUsuario):
 
     rol_usuario = forms.ChoiceField(choices=ROLES_USUARIO, label='Tipo de usuario', required=True)
 
+    def generate_user_password(self):
+        return self.instance.first_name+'_'+self.instance.last_name
+
     def save(self, *args, **kwargs):
         """ Override save method to add group to the user.
 
         This overrides the save method of forms.ModelForm so that we can
         add the corresponding group to the user that is being created.
         The group is chosen based on the ChoiceField defined above
         """
-        user = super(FormaCreacionUsuario, self).save(*args, **kwargs)
         data = self.cleaned_data
-        if data['rol_usuario'] == CAPTURISTA_GROUP:
-            # capturista's group is added in the save method of the Model.
-            capturista = Capturista(user=user)
-            capturista.save()
+        # Create user
+        if self.instance.pk is None:
+            user = super(UserForm, self).save(*args, **kwargs)
+            # TODO: Change the password generation for an PasswordResetForm
```

Comment:
I figured it was another branch with its own tests, this is not functionality for editing the user